corehq/apps/reports/tests/test_project_health.py

```diff
@@ -0,0 +1,138 @@
+from django.test import TestCase
+import datetime
+from dimagi.utils.dates import add_months
+from corehq.apps.reports.standard.project_health import MonthlyPerformanceSummary
+from corehq.apps.data_analytics.models import MALTRow
+from corehq.const import MISSING_APP_ID
+from corehq.apps.domain.models import Domain
+from corehq.apps.users.models import CommCareUser
+
+
+class MonthlyPerformanceSummaryTests(TestCase):
+    DOMAIN_NAME = "test"
+    USERNAME = "testuser"
+    USERNAME1 = "testuser1"
+    USERNAME2 = "testuser2"
+    USER_TYPE = "CommCareUser"
+    now = datetime.datetime.utcnow()
+    year, month = add_months(now.year, now.month, 1)
+    month_as_date = datetime.date(year, month, 1)
+    prev_month_as_date = datetime.date(year, month - 1, 1)
+
+    @classmethod
+    def setUpClass(cls):
+        super(MonthlyPerformanceSummaryTests, cls).setUpClass()
+        cls._setup_domain()
+        cls._setup_users()
+        cls._setup_malt_tables()
+        users_in_group = []
+        cls.prev_month = MonthlyPerformanceSummary(
+            domain=cls.DOMAIN_NAME,
+            performance_threshold=15,
+            month=cls.prev_month_as_date,
+            previous_summary=None,
+            users=users_in_group,
+            has_group_filter=False,
+        )
+        cls.month = MonthlyPerformanceSummary(
+            domain=cls.DOMAIN_NAME,
+            performance_threshold=15,
+            month=cls.month_as_date,
+            previous_summary=cls.prev_month,
+            users=users_in_group,
+            has_group_filter=False,
+        )
+
+    @classmethod
+    def tearDownClass(cls):
+        users = CommCareUser.all()
+        for user in users:
+            user.delete()
+        MALTRow.objects.all().delete()
```

Comment:
This one also needs a `super` call, but that can be done in a separate PR so we don't have to wait on tests for that change (it's needed for the upgrade to 1.8, but it won't cause any immediate failures).